.mailmap

```diff
@@ -1211,6 +1220,7 @@ Zeel Shah <kshah215@gmail.com>
 Zhi-Qiang Zhou <zzq_890709@hotmail.com> zhouzq-thu <zzq_890709@hotmail.com>
 Zhongshi <zj495@nyu.edu>
 Zlatan Vasović <zlatanvasovic@gmail.com>
+Zouhair <zouhair.mahboubi@gmail.com>
```

Comment:
@zouhairm how do you want your name to be recorded in the AUTHORS file?

Reply:
yes fine with that.